action.yml

```diff
@@ -33,8 +33,11 @@ runs:
       shell: bash
       env:
         MIX_ENV: test
-      
+
     - name: Install Node Dependencies
+      env: 
+        VARIANT_WITHOUT_NODE: '["api"]'
```

Comment:
Hi @hanam1ni should we revert this ENV a bit to `VARIANT_WITH_NODE` to make it easy to understand 🤔 IMO or we could change to another name like

```
VARIANT_WITH_NODE_DEPENDENCY: '["web", "live"]'

if: contains(fromJson(env.VARIANT_WITH_NODE_DEPENDENCY), inputs.variant) == true
```